### PR TITLE
Feat: `default` file to avoid credentials commits

### PR DESCRIPTION
**Pull request by mbologna**
This PR adds placeholder files (*.default) for config files, in order to avoid accidental leakage of credentials.

### CODE REVIEW

**Comment by moio** [comment]
Please add a description to this PR, the effect is not clear from the title. Also please modify the README to update instructions for new users.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pillar/config.sls
```
Comment (moio) [inline .gitignore]: Last question: why is this here? This PR does not seem to affect this file at all. I guess this line should just be removed.

**Review by moio — CHANGES_REQUESTED** [review]

**Review by mbologna — COMMENTED** [review]
Why do we need .default files in openvpn config on raspberry side?

File: .gitignore (hunk 1 above)
Comment (mbologna) [reply .gitignore]: You're right. Removed.

**Review by moio — APPROVED** [review]

**Pull request merged**